google-cloud-spanner/src/test/java/com/google/cloud/spanner/connection/StatementTimeoutTest.java

```diff
@@ -1133,8 +1197,9 @@ public void testTimeoutExceptionDdlBatch() {
       // the following statement will NOT timeout as the statement is only buffered locally
       connection.execute(Statement.of(SLOW_DDL));
       // the commit sends the statement to the server and should timeout
-      expected.expect(SpannerExceptionMatcher.matchCode(ErrorCode.DEADLINE_EXCEEDED));
       connection.runBatch();
```

Comment:
Add a `fail("Expected exception")` to prevent the test case from succeeding in case of no error.

Reply:
Done.